modules/hibernating-cluster-hibernate.adoc

```diff
@@ -0,0 +1,97 @@
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                      STATUS  ROLES                 AGE   VERSION
+ci-ln-812tb4k-72292-8bcj7-master-0        Ready	  control-plane,master  32m   v1.30.7
+ci-ln-812tb4k-72292-8bcj7-master-1        Ready	  control-plane,master  32m   v1.30.7
+ci-ln-812tb4k-72292-8bcj7-master-2        Ready	  control-plane,master  32m   v1.30.7
+Ci-ln-812tb4k-72292-8bcj7-worker-a-zhdvk  Ready	  worker                19m   v1.30.7
+ci-ln-812tb4k-72292-8bcj7-worker-b-9hrmv  Ready	  worker                19m   v1.30.7
+ci-ln-812tb4k-72292-8bcj7-worker-c-q8mw2  Ready	  worker                19m   v1.30.7
+----
++
+All nodes should show `Ready` in the `STATUS` column.
+
+. Ensure that all cluster Operators are in a good state by running the following command:
++
+[source,terminal]
+----
+$ oc get clusteroperators
+----
++
+.Example output
+[source,terminal]
+----
+NAME                      VERSION                             AVAILABLE  PROGRESSING  DEGRADED  SINCE   MESSAGE
+authentication            4.17.0-0.nightly-2024-12-19-020614  True       False        False     51m
+baremetal                 4.17.0-0.nightly-2024-12-19-020614  True       False        False     72m
+cloud-controller-manager  4.17.0-0.nightly-2024-12-19-020614  True       False        False     75m
+cloud-credential          4.17.0-0.nightly-2024-12-19-020614  True       False        False     77m
+cluster-api               4.17.0-0.nightly-2024-12-19-020614  True       False        False     42m
+cluster-autoscaler        4.17.0-0.nightly-2024-12-19-020614  True       False        False     72m
+config-operator           4.17.0-0.nightly-2024-12-19-020614  True       False        False     72m
+console                   4.17.0-0.nightly-2024-12-19-020614  True       False        False     55m
+...
+----
++
+All cluster Operators should show `AVAILABLE`=`True`, `PROGRESSING`=`False`, and `DEGRADED`=`False`.
+
+. Ensure that all machine config pools are in a good state by running the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
+[source,terminal]
+----
+NAME    CONFIG                                            UPDATED  UPDATING  DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT  DEGRADEDMACHINECOUNT  AGE
+master  rendered-master-87871f187930e67233c837e1d07f49c7  True     False     False     3             3                  3                    0                     96m
+worker  rendered-worker-3c4c459dc5d90017983d7e72928b8aed  True     False     False     3             3                  3                    0                     96m
+----
++
+All machine config pools should show `UPDATING`=`False` and `DEGRADED`=`False`.
+
+. Stop the cluster virtual machines:
++
+Use the tools native to your cluster's cloud environment to shut down the cluster's virtual machines.
```

Comment:
Could this just a single para?
```suggestion
. Stop the cluster virtual machines. Use the tools native to your cluster's cloud environment to shut down the cluster's virtual machines.
```
or at least drop the colon?
```suggestion
. Stop the cluster virtual machines.
+
Use the tools native to your cluster's cloud environment to shut down the cluster's virtual machines.
```

Reply:
You know, I played around with having these all on one line, and they looked a bit too cluttered. I'll take another look though.